apps/api/src/donations/donations.service.ts

```diff
@@ -42,8 +42,7 @@ export class DonationsService {
     private exportService: ExportService,
   ) {}
 
-  private cache = new HotCache()
-  private cacheTimeout = 30 * 1000 // 30 seconds
+  private cache = new HotCache(this.config.get<number>('hotcache.ttl') || 0, !!this.config.get<boolean>('hotcache.enabled'))
```

Comment:
Is it possible to inject the ConfigService, inside the `HotCache` class and get the configs from there, instead of getting them on every creation of instance?